CRM/Core/ManagedEntities.php

```diff
@@ -667,11 +667,11 @@ protected function loadDeclarations(): void {
 
   protected function loadManagedEntityActions(): void {
     $managedEntities = Managed::get(FALSE)->addSelect('*')->execute();
+    $this->managedActions = [];
```

Comment:
Unit test revealed this variable was leaky. The purpose of this function is to rebuild this array, but it needs to be reset first due to the singleton pattern.